src/main/kotlin/com/autonomousapps/extension/AbiHandler.kt

```diff
@@ -40,11 +42,29 @@ public abstract class AbiHandler @Inject constructor(objects: ObjectFactory) {
 /** @see [AbiHandler]. */
 public abstract class ExclusionsHandler @Inject constructor(objects: ObjectFactory) {
 
+  internal val classInclusions = objects.setProperty(String::class.java).convention(emptySet())
+  internal val annotationInclusions = objects.setProperty(String::class.java).convention(emptySet())
   internal val classExclusions = objects.setProperty(String::class.java).convention(emptySet())
   internal val annotationExclusions = objects.setProperty(String::class.java).convention(emptySet())
   internal val pathExclusions = objects.setProperty(String::class.java).convention(emptySet())
   internal val excludedSourceSets = objects.setProperty(String::class.java).convention(emptySet())
 
+  /**
+   * Include given classes into ABI analysis, which means that any class that does not match the given [classRegexes] regex
+   * will not be considered as part of the module's ABI.
+   */
+  public fun includeClasses(@Language("RegExp") vararg classRegexes: String) {
+    classInclusions.addAll(*classRegexes)
+  }
+
+  /**
+   * Include any class into ABI analysis that is annotated with annotations that match [annotationRegexes], which means
+   * those classes will be considered as part of the module's ABI.
+   */
+  public fun includeAnnotations(@Language("RegExp") vararg annotationRegexes: String) {
```

Comment:
Is the inverse also true (as above), which is that once an inclusion is set, anything that doesn't match _isn't_ considered part of the ABI?